html/pfappserver/root/config/trafficshaping/list.tt

```diff
@@ -0,0 +1,57 @@
+      <table id="items" class="table items" data-list-uri="[% c.uri_for(c.controller.action_for('list')) %]">
+      [% IF items.size %]
+        <thead>
+          <tr>
+            <th>[% l('TrafficShaping Name') %]</th>
+            <th>[% l('Description') %]</th>
+            <th></th>
+          </tr>
+        </thead>
+      [% END %]
+        <tbody>
+          [% IF items.size %]
+              [% FOREACH item IN items %]
+              <tr>
+                <td class="item"><a href="[% c.uri_for(c.controller.action_for('view'), [ item.id ]) %]">[% item.id | html %]</a></td>
+                <td>[% item.description | html%]</td>
+                <td>
+                  [% IF can_access("TRAFFIC_SHAPING_DELETE") %]<a class="btn btn-mini btn-danger" href="[% c.uri_for(c.controller.action_for('remove'), [ item.id ]) %]">[% l('Delete') %]</a>[% END %]
+                </td>
+              </tr>
+              [% END -%]
+          [% ELSE %]
+              <tr> 
+                <td colspan=3>
+                  <div id="noItems" class="unwell">
+                    <i class="icon-unlock icon-large"></i><p>[% l('No Traffic Shaping defined') %]</p>
```

Comment:
Instead of calling it "Traffic Shaping" everywhere like if it was an object, it would be better to call it "Traffic shaping policy"